docs/blog/2019-07-03-using-themes-for-distributed-docs/index.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Using Themes for Distributed Docs
+date: 2019-07-03
+author: Trevor Blades
+excerpt: "Build a full-featured Gatsby website one time, and apply it to multiple data sources."
+tags:
+  - themes
+  - mdx
+---
+
+Gatsby is a great tool for building static websites, and offers a lot of value when building pages based on a set of Markdown files. This makes it an ideal partner for building documentation websites, which are typically built statically, and updated by writing Markdown.
+
+At [Apollo](https://www.apollographql.com), our documentation spans multiple repositories. This is good because it keeps different codebases more tightly coupled with their documentation, issues, and PRs. However, when it comes to building a documentation website, this presents us with a challenge:
+
+> How should we bring all of our documentation together into one consistent experience?
+
+Using Gatsby, it would be possible to source files from each repository (using `gatsby-source-git`, for example) and deploy one mega-site. This would satisfy our need to tie all of our repos’ documentation together, but it would also mean that any change in one repo would mean building, testing, and deploying the entire website.
+
+![Many sources going through one website](./images/many-to-one.png)
+
+An alternative approach would be for each repo to deploy a piece of the website based on some shared configuration, allowing each of them to exist independently but remain up-to-date with changes to the website layout and infrastructure. In this world, changes to one repo’s documentation only means rebuilding and deploying its small part of the greater website. The goal of this approach is to reduce or remove the bottlenecks involved in making docs changes.
+
+![Each source has its own website](./images/one-to-one.png)
+
+Gatsby Themes are a great choice for this task. Themes allow us to build a full-featured Gatsby website—configuring plugins, writing custom components, using GraphQL queries—one time and apply it to multiple data sources. Themes are highly configurable, so each website can have its own defining features, like a page title, description, and more.
+
+In this blog post, I’m going to talk about some of our favourite parts about using Gatsby Themes to deliver a consistent docs experience.
```

Comment:
```suggestion
In this blog post, I’m going to talk about some of our favorite parts about using Gatsby Themes to deliver a consistent docs experience.
```
Updating per the [Gatsby Style Guide](https://www.gatsbyjs.org/contributing/gatsby-style-guide/#prefer-us-english).